commcare_connect/templates/tailwind/components/cards/activity_card.html

```diff
@@ -1,26 +1,26 @@
-{% load static i18n %}
 <div class="activity-comp flex flex-col p-6 rounded-lg bg-white gap-5">
     <div class="flex gap-4 items-center">
-        <i class="fa-light {{ icon }}"></i>
+        <i class="fa-light fa-{{ icon }}"></i>
         <p class="text-base">
-            {{ title }}
+            {{ name }}
```

Comment:
You will need to undo these changes that updates Python vars. Please see how this is referred to in home.html
```
 {% include 'tailwind/components/cards/activity_card.html' with title="Visits Pending Review" rows=review_counts icon="fa-clock-rotate-left" %}
```